### DISCUSSION TITLE
Get/Set Notes for studies

### DISCUSSION THREAD

**Discussion by turbotimon** [optuna/optuna#5361] [Q&A] [answered]
**What I'm looking for:**
I'm looking for a way to add additional information to a study, e.g. what features where used.

**What could be the solution:**
So I discovered, that in optuna dashboard, there is the posibility to add a note (e.g. under `http://127.0.0.1:8080/dashboard/studies/1/note`). However, it seems that you can not create or load them over the API?

**What I expected:**
```
create_study(
    note="some note text"
)

study = load_study(...)
study.note # get the note
```

Is this feature planed or are there other possibilities?

**Comment by nzw0301** [comment, marked as answer]
How about https://optuna.readthedocs.io/en/stable/tutorial/20_recipes/003_attributes.html?

**Reply by turbotimon** [reply]
Ah yes! Exactly what i was looking for. I was aware that user attr exists on trials, but not on studies. Thank you very much ❤️
